tox.ini

```diff
@@ -153,4 +153,4 @@ basepython=python
 [flake8]
 ignore=W391,E231,E201,E202,E203,E261,E302,E128,E126,E124
 max-line-length=120
-exclude = tests
+exclude = tests
```

Comment:
add this config to sublime so that things like this don't sneak into your diff `ensure_newline_at_eof_on_save` http://docs.sublimetext.info/en/latest/reference/settings.html